packages/react/src/app-layout/AppLayoutHeader.stories.tsx

```diff
@@ -103,3 +104,35 @@ export const WithAccountLinkLongName: Story = {
 		},
 	},
 };
+
+export const CoBranding: Story = {
+	args: {
+		accountDetails: {
+			name: exampleData.userNames.long,
+			secondaryText: exampleData.businessNames.regular[0],
+			href: '#',
+		},
+		secondHref: '/',
+		secondLogo: <AISLogo />,
+	},
+};
+
+export const CoBrandingDividerPositionBetween: Story = {
+	args: {
+		dividerPosition: 'between',
+		secondHref: '/',
+		secondLogo: <AISLogo />,
+	},
+};
+
+export const CoBrandingWithAccountDetails: Story = {
```

Comment:
This is the same as CoBranding

Reply:
- [ ] remove accountDetails from CoBranding